requirements-dev.txt

```diff
@@ -6,3 +6,4 @@ responses>=0.25
 ruff==0.1.14
 setuptools==69.0.3
 setuptools-scm==8.0.4
+pandas
```

Comment:
No version pin here bc for our purposes, it doesn't matter, and because the current release of pandas isn't available for Python 3.8.